source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -282,6 +282,108 @@ the Query API:
      }
    ]
 
+Function Operations
+~~~~~~~~~~~~~~~~~~~
+
+You can apply custom functions to values of type
+``MqlValue`` using the methods described in this section.
+
+To improve readability and brevity in your code, you can move redundant code into
+static methods. However, it is often not possible to chain these methods in
+Java. The ``passTo()`` method lets you chain custom methods in your code.
```

Comment:
Maxim suggested including this because there isn't really an equivalent in the Query API and the way this method works is pretty different than all the others. The purpose is also different than the others, in a way